### PR TITLE
chore: fix broken import paths in nested docs plugin README 

### LINKED ISSUE #4365: Incorrect imports in plugin-nested-docs readme
# Documentation Issue

In the plugin-nested-docs readme, there's some sample code with the following imports:
```
import createParentField from "@payloadcms/plugin-nested-docs/fields/parent";
import createBreadcrumbsField from "@payloadcms/plugin-nested-docs/fields/breadcrumbs";
````

Using the most recent version of that plugin, it seems that createParentField and createBreadcrumbsField can be exported either from `@payloadcms/plugin-nested-docs/fields` or from `@payloadcms/plugin-nested-docs/dist/fields/parent` and `@payloadcms/plugin-nested-docs/dist/fields/breadcrumbs`, respectively.

It's possible I'm misunderstanding something, but if I am not, I believe those code samples may need to be updated.

### PR DESCRIPTION
**Pull request by JessRynkar**
## Description

Closes #4365 

Nested docs plugin README has outdated import paths.

- [X] I have read and understand the [CONTRIBUTING.md](https://github.com/payloadcms/payload/blob/main/CONTRIBUTING.md) document in this repository.

## Type of change

- [X] Chore (non-breaking change which does not add functionality)

### CODE REVIEW

File: packages/plugin-nested-docs/README.md [hunk 1]
```diff
@@ -135,8 +135,8 @@ You can also extend the built-in `parent` and `breadcrumbs` fields per collectio
 
 ```js
 import { CollectionConfig } from "payload/types";
-import createParentField from "@payloadcms/plugin-nested-docs/fields/parent";
-import createBreadcrumbsField from "@payloadcms/plugin-nested-docs/fields/breadcrumbs";
+import createParentField from "@payloadcms/plugin-nested-docs/src/fields/parent";
```
Comment (jacobsfletch) [inline packages/plugin-nested-docs/README.md:138]: This is not right, we should not be importing from `src`. If this export is no longer available it is likely due to a change in the publish config post-monorepo.

**Comment by JessChowdhury** [comment]
@jacobsfletch updated to the correct paths.

File: packages/plugin-nested-docs/README.md [hunk 2]
```diff
@@ -135,8 +135,8 @@ You can also extend the built-in `parent` and `breadcrumbs` fields per collectio
 
 ```js
 import { CollectionConfig } from "payload/types";
-import createParentField from "@payloadcms/plugin-nested-docs/fields/parent";
-import createBreadcrumbsField from "@payloadcms/plugin-nested-docs/fields/breadcrumbs";
+import createParentField from "@payloadcms/plugin-nested-docs/fields";
+import createBreadcrumbsField from "@payloadcms/plugin-nested-docs/fields";
```
Comment (jacobsfletch) [inline packages/plugin-nested-docs/README.md:139]: This is still not correct, there cannot be two default exports from the same file.

Comment (jacobsfletch) [reply packages/plugin-nested-docs/README.md:139]: ^^ I've got these imports sorted out in #4462

**Comment by jacobsfletch** [comment]
Superseded by #4462

**Pull request closed without merging**